core/gateway.go

```diff
@@ -79,7 +86,7 @@ func (g *GatewayClient) CombinedResult(ctx context.Context) (*bkgw.Result, error
 		}
 		mergeInputs = append(mergeInputs, state)
 	}
-	llbdef, err := llb.Merge(mergeInputs).Marshal(ctx)
+	llbdef, err := llb.Merge(mergeInputs, llb.WithCustomName("combined session result")).Marshal(ctx)
```

Comment:
Handy, that name could get pretty long haha

Reply:
Yeah I noticed that too... luckily this codepath doesn't even run anymore when using the cache service, it's only needed for the upstream remote cache providers.